docs/source/usage.rst

```diff
@@ -814,7 +814,7 @@ Example:
 
 OmegaConf.select
 ^^^^^^^^^^^^^^^^
-OmegaConf.select() allow you to select a config node or value using a dot-notation key.
+OmegaConf.select() allow you to select a config node or value, using either a dot-notation or brackets to denote sub-keys.
```

Comment:
Can you mention it in the `Config node interpolation` as well?

Reply:
Done in d7b7b00